_guides/defining_execution_frequency_for_flows.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Defining execution frequency for flows
+layout: article
+section: Building integration flows
+category: component
+order: 4
+since: 20181218
+---
+
+Scheduling tasks are part of the {{site.data.tenant.name}} functionality. Every flow is executed every 3 minutes which is Platform's default setting. Depending on the business needs, that default setting might not always be suitable. To proceed further, you may take advantage of some of the flow scheduling features that are going to be described in this article.
+
+## The Scheduling
+
+Please be aware, that changing the flow's schedule means modifying the flow. Here some of the requirements to follow: 
+
+*   For modifying a flow, you should have specific permissions.
+*   As soon as you modify the flow's schedule, you should restart it to let the System apply the changes.
+
+For more information regarding the CRON scheduling, please follow the steps below:
+
+### Navigation
+
+Create a [flow](#realtime-flows) or pick the existing one by navigating to the *Workspace > Flows > Choose the necessary Flow > Settings* tab. See the screenshot below.
+
+![CRON Navigation](/assets/img/developer-guide/cron/cron_navigation.png "CRON Navigation")
+
+### Working with Expressions
+
+The *Scheduling tab* consists of the *Cron Expression* section and the *Next Occurrences* one. 
+
+Specify the necessary value into the *Minute field* that is under the *Cron Expression* title. For example, we want to restart the flow every minute, so we type in the following `*/1` value. The *Next Occurrences* section represents the expected result. Check the screenshot below.
+
+![Scheduling](/assets/img/developer-guide/cron/scheduling.png "Scheduling")
+
+> **Please Note**: By default, the *Cron Expression* section's value is always in the *GMT* format. The *Next Occurrences* section converts the specified GMT value into your Local Time format one. This way it demonstrates the expected result, so you can make sure it is correct.
+
+In case of mistake or wrong value, the System displays an Error Message.
+
+![Scheduling Error](/assets/img/developer-guide/cron/scheduling_error.png "Scheduling Error")
+
+
+Below are some predefined scheduling definitions:
+
+| Description                                                | Equal to        |
```

Comment:
Equal to => CRON expression

Reply:
I would also flip the columns order so that you have the expression on the left and the description on the right